src/swap/saga.ts

```diff
@@ -352,7 +352,7 @@ export function* swapSubmitSaga(action: PayloadAction<SwapInfo>) {
       throw new Error(`Swap transaction reverted: ${swapTxReceipt.transactionHash}`)
     }
 
-    yield* put(swapSuccess(swapId))
+    yield* put(swapSuccess({ swapId, fromTokenId, toTokenId }))
```

Comment:
@dievazqu yes, we decided to ignore failed swaps for now. because if the swap failed, something must be wrong. and it's probably okay to keep a bit of friction for the user in recreating it over again.